src/main/java/com/faforever/client/map/CommentCardController.java

```diff
@@ -0,0 +1,32 @@
+package com.faforever.client.map;
+
+
+import com.faforever.client.legacy.map.Comment;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.GridPane;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class CommentCardController {
+
+  @FXML
+  Label authorLabel;
+  @FXML
+  Label dateLabel;
+  @FXML
+  Label commentLabel;
+  @FXML
+  GridPane root;
+
+  public GridPane getRoot() {
+    return root;
+  }
+
+  public void createComment(Comment comment) {
+    authorLabel.setText(comment.getAuthor());
+    dateLabel.setText(comment.getDate().format(DateTimeFormatter.ofPattern("MMMM dd yyyy", Locale.US)));
```

Comment:
This defies i18n. Use `i18n.get("datePattern")` instead (which I just realized uses the wrong format.. I vote for `uuuu-MM-dd` which is the only sane format and ISO standard (ISO 8601)